### PR TITLE
Prepare for 0.20.3 and fork mustache4dart into Dartdoc

### PR DESCRIPTION
**Pull request by jcollins-g**
Looks like it will be a while before mustache4dart is ready to declare it is ready for 2.0 users.  But dartdoc needs to work on Dart 2.  So, fork.

FYI valotas/mustache4dart#78

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,16 @@ language: dart
 sudo: false
 dart:
   - "dev/raw/latest"
+  - stable
```
Comment (devoncarew) [inline .travis.yml]: 👍

**Comment by chalin** [comment]
How is this coming along?

**Comment by kevmoo** [comment]
@jcollins-g – looks like the flutter failures is just due to `2.0.0` being the version...

**Comment by jcollins-g** [comment]
dartdoc should probably be using the SDK inside Flutter to run pub/dart.  fix coming up.

**Comment by kevmoo** [comment]
ack

**Comment by chalin** [comment]
@jcollins-g - can you let me know once the new version (0.20.3?) makes it to pub?

**Comment by jcollins-g** [comment]
will do.  This PR turns out to be unpublishable because pub really doesn't like path dependencies, even ones which are safe (contained to the package).

**Comment by chalin** [comment]
Ah, not only is it unpublishable, it is unusable from `master` (I was hoping otherwise):

```console
> pub global activate --source git https://github.com/dart-lang/dartdoc.git
Package dartdoc is currently active at version 0.20.1.
Resolving dependencies... (4.2s)
Error on line 24, column 25: Invalid description: "third_party/pkg/mustache4dart" is a relative path, but this isn't a local pubspec.
  mustache4dart: {path: 'third_party/pkg/mustache4dart'}
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

cc @kwalrath

**Comment by jcollins-g** [comment]
You can actually use it from master, you just have to check it out yourself
and run it through pub global activate -spath




Janice

**Comment by chalin** [comment]
That works, thanks!

**Pull request merged**